airflow/sensors/time_delta.py

```diff
@@ -59,28 +59,34 @@ class TimeDeltaSensorAsync(TimeDeltaSensor):
     Will defers itself to avoid taking up a worker slot while it is waiting.
 
     :param delta: time length to wait after the data interval before succeeding.
+    :param end_from_trigger: End the task directly from the triggerer without going into the worker.
 
     .. seealso::
         For more information on how to use this sensor, take a look at the guide:
         :ref:`howto/operator:TimeDeltaSensorAsync`
 
     """
 
+    def __init__(self, *, end_from_trigger: bool = False, delta, **kwargs):
+        super().__init__(delta=delta, **kwargs)
+        self.end_from_trigger = end_from_trigger
+        self.delta = delta
```

Comment:
```suggestion
        self.delta = delta
```

I think it's assigned in `TimeDeltaSensor`

Reply:
Changed it